doc/sphinx/Pacemaker_Python_API/pacemaker.rst

```diff
@@ -0,0 +1,9 @@
+Pacemaker
+=========
+
+The pacemaker module re-exports its submodules, so code should import directly from
```

Comment:
It does? I honestly don't know that much about Python export/import schemes. It looks like the pacemaker module just imports `BuildOptions` from `pacemaker.buildoptions`. So it's not really re-exporting the submodule; it's importing (and thus indirectly exporting) a single class (which happens to be the only class) from a submodule.

Again I have only a tenuous grasp of this so I'm not taking a strong stance here. The "best practice" is clear enough, and this can always be updated later if our public Python code becomes non-trivial.

Reply:
We could reword this, because you're correct - it really is just re-exporting those two things and if we add more stuff, we'll have to add more to those lines.  We could also `from pacemaker.buildoptions import *` but we'd also have to tell pylint to ignore that.  It's not a big deal, though.